<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryWidget</class>
 <widget class="QWidget" name="HistoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ÂéÜÂè≤ËÆ∞ÂΩï</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#HistoryWidget {
    background-color: #f5f7fa;
}

QLabel#titleLabel {
    font-size: 20px;
    font-weight: bold;
    color: #2c3e50;
}

QGroupBox {
    font-size: 14px;
    font-weight: bold;
    color: #2c3e50;
    border: 1px solid #dcdfe6;
    border-radius: 8px;
    margin-top: 12px;
    padding-top: 10px;
    background-color: white;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 15px;
    padding: 0 8px;
}

QTableWidget {
    border: none;
    background-color: white;
    gridline-color: #ebeef5;
    font-size: 12px;
}

QTableWidget::item {
    padding: 8px;
}

QTableWidget::item:selected {
    background-color: #ecf5ff;
    color: #409eff;
}

QHeaderView::section {
    background-color: #f5f7fa;
    padding: 10px;
    border: none;
    border-bottom: 1px solid #ebeef5;
    font-weight: bold;
    color: #606266;
}

QPushButton {
    background-color: #409eff;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 8px 16px;
    font-size: 13px;
}

QPushButton:hover {
    background-color: #66b1ff;
}

QPushButton:pressed {
    background-color: #3a8ee6;
}

QPushButton#exportBtn {
    background-color: #67c23a;
}

QPushButton#exportBtn:hover {
    background-color: #85ce61;
}

QPushButton#clearBtn {
    background-color: #f56c6c;
}

QPushButton#clearBtn:hover {
    background-color: #f78989;
}

QDateEdit {
    border: 1px solid #dcdfe6;
    border-radius: 4px;
    padding: 6px 10px;
    background-color: white;
}

QComboBox {
    border: 1px solid #dcdfe6;
    border-radius: 4px;
    padding: 6px 10px;
    background-color: white;
    min-width: 120px;
}

QComboBox::drop-down {
    border: none;
    width: 20px;
}

QComboBox::down-arrow {
    width: 12px;
    height: 12px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>üìú ÂéÜÂè≤ËÆ∞ÂΩï</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="headerSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="cityLabel">
       <property name="text">
        <string>ÂΩìÂâçÂüéÂ∏ÇÔºö--</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #606266; font-size: 14px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="filterGroup">
     <property name="title">
      <string>Êü•ËØ¢Êù°‰ª∂</string>
     </property>
     <layout class="QHBoxLayout" name="filterLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="startLabel">
        <property name="text">
         <string>ÂºÄÂßãÊó•ÊúüÔºö</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="startDateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="endLabel">
        <property name="text">
         <string>ÁªìÊùüÊó•ÊúüÔºö</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="endDateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="typeLabel">
        <property name="text">
         <string>Êï∞ÊçÆÁ±ªÂûãÔºö</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="typeComboBox">
        <item>
         <property name="text">
          <string>ÂÖ®ÈÉ®</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ÂÆûÊó∂Â§©Ê∞î</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Â§©Ê∞îÈ¢ÑÊä•</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="queryBtn">
        <property name="text">
         <string>Êü•ËØ¢</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="filterSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dataGroup">
     <property name="title">
      <string>ÂéÜÂè≤Êï∞ÊçÆ</string>
     </property>
     <layout class="QVBoxLayout" name="dataLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QTableWidget" name="historyTable">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Êó•Êúü</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Á±ªÂûã</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ê∏©Â∫¶</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Â§©Ê∞î</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ÊπøÂ∫¶</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>È£éÈÄü</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ê∞îÂéã</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actionLayout">
     <item>
      <widget class="QLabel" name="countLabel">
       <property name="text">
        <string>ÂÖ± 0 Êù°ËÆ∞ÂΩï</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #909399; font-size: 12px;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="actionSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exportBtn">
       <property name="text">
        <string>ÂØºÂá∫Êï∞ÊçÆ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearBtn">
       <property name="text">
        <string>Ê∏ÖÁ©∫ËÆ∞ÂΩï</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
